index.html

```diff
@@ -0,0 +1,101 @@
+                        </ul>
+                    <li>RSSchool lectures on working:</li>
+                        <ul>
+                            <li>Git</li>
+                            <li>Basics of JS</li>
+                        </ul>
+                    <li>Vitaliy Fridman's workshop “New Adventures in Front-End”   September 2018</li>
+                </ol>
+            </p>
+        </div>
+        <div> 
+            <h3>WORK EXPERIENCE</h3>
+            <p>Belarusian State Institute of Metrology<br>
+               ENGINEER OF METROLOGY     August 2007 - Present
+            </p>
+            <p>Working with precise equipment, determinating if it works correct in accordance to its documentation 
+               as well as development and writing documents of accuracy caracteristics based on research work.
+            </p>
+            <p><strong>Duties</strong>
+                <ul>
+                    <li>Verification and calibration of measuring equipment</li>
+                    <li>Development of technical normative legal acts for verification and calibration</li>
+                    <li>Development and study of new methods and measuring instruments, reference materials</li>
+                    <li>Carrying out fundamental and applied scientific research in optics</li>
+                    <li>Working with cusomers and service engineers to ensure process the work process</li>
+                </ul>
+            </p>
+        </div>
+        <div>
+            <h3>SKILLS AND COMPETENCIES</h3>
+            <p>
+                <ul>
+                    <li>Good mathematical and logical abilities</li>
+                    <li>Understanding of principles of work of Distributed Version Control System (DVCS) Git</li>
+                    <li>Basic knowledge of JS and object oriented programming</li>
+                    <li>Good administration and organisational skills</li>
+                    <li>Having a logical approach to problem solving</li>
+                    <li>Responsible and with strong attention to detail</li>
+                </ul>
+            </p>
+        </div>
+        <div>
+            <h3>DEVELOPING EXPERIENCE</h3>
+            <p><strong>Examples of my code:</strong></p>
+            <ol>
+                <li><a href = "https://github.com/svyatlo/Money-Exchange/blob/master/src/index.js">
+                    Number of coins needed to make change a given amount of currency</a></li>
+                <li><a href = "https://github.com/svyatlo/advanced-zeros/blob/master/src/index.js">
+                    Martians factorials!</a></li>
+                <li><a href = "https://www.codewars.com/users/svyatlo/completed_solutions">
+                    CodeWars solutions.</a></li>
+            </ol>
+        </div>
+        <div>
+            <h3 class = "info">PERSONAL INFORMATION</h3>
+            <p>phone: +375445346695<br>
+               e-mail: svyatlo@list.ru
+            </p>
```

Comment:
```
<p>phone: +375445346695<br>
   e-mail: svyatlo@list.ru
```
You could wrap phone and email in tag <a> with specific href

```
<a href="tel:+375445346695">+375445346695</a>
<a href="mailto:svyatlo@list.ru>svyatlo@list.ru</a>
```
to add more usability for users